rpc/documentation/api.md

```diff
@@ -774,6 +779,19 @@ results of a single ticket.
 
     - `REVOKED`: A ticket that has been revoked.
 
+- `BlockMeta block`: The block the ticket was mined
```

Comment:
should document that it is `null` if unmined

Reply:
It's not null, because the call stack goes all the way to this: https://github.com/decred/dcrwallet/blob/master/wallet/udb/txquery.go#L108-L111

Height gets defined as -1, but the other stuff is undefined (so, technically a zero value). That's why I documented on the individual fields as "undefined".